notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<p>ENDPOINT = "us-central1- ..."  -&gt;</p><p>ENDPOINT = f"{REGION}-..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>actually that definition of ENDPOINT is redundant (see previous cell) and unused here so I think I can just delete that line.</p>